apps/mobile/src/libs/components/wheel/wheel.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { getRadiansFromPercent } from "shared";
+
+import { Defs, Stop, Svg, SvgGradient } from "~/libs/components/components";
+import { BaseColor } from "~/libs/enums/enums";
+import { type WheelDataItem } from "~/libs/types/types";
+
+import { AnimatedSector, LabelOnCircle } from "./libs/components/components";
+import { WheelSetting } from "./libs/enums/enums";
+import { getWheelCategoryParameters } from "./libs/helpers/helpers";
+
+type Properties = {
+	categoriesData: WheelDataItem[];
+	isLabelShown?: boolean;
+	maxScore: number;
+	size: number;
+};
+
+const Wheel: React.FC<Properties> = ({
+	categoriesData,
+	isLabelShown = true,
+	maxScore,
+	size,
+}: Properties) => {
+	const MINIFY_TWICE_COEFFICIENT = 0.5;
+	const ANIMATION_TIME = 500;
```

Comment:
The same here - put constants outside

Reply:
May I put constants from all Wheel-related components into Wheel constants file? If so, is it allowed to list and export all those in wheel/libs/constants/constants.ts ?

Reply:
You don't need to put them in a separate file. Just declare them in the same file but before the component (below the `type Properties...`)

Reply:
Also, please don't resolve conversation if you have some unclosed questions or didn't apply the changes.